packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -357,7 +368,9 @@ const ReleaseDetailsBody = () => {
     isError: isReleaseError,
     error: releaseError,
   } = useGetReleaseQuery({ id: releaseId });
+
   const release = releaseData?.data;
+  const selectedGroupBy = query?.groupBy || 'contentType';
```

Comment:
this var right now it any, can we explicit the type as 
`const selectedGroupBy: (typeof GROUP_BY_OPTIONS)[number] = query?.groupBy || 'contentType';`

Reply:
The inferred type seems to be coming through as `ReleaseActionGroupBy` on my end 🤔 
<img width="506" alt="Screenshot 2024-01-08 at 17 15 44" src="https://github.com/strapi/strapi/assets/26598053/20dd2659-f3ac-46e6-b5b3-2cb184f836c5">

Reply:
ok, for me was an any but if you have already the inferred type that's ok for me